### ISSUE TITLE
Cannot alter image names of media items

### ISSUE THREAD

**Issue by joecoolle** [Simon-Initiative/oli-torus#1101]
**Describe the bug**
I cannot upload image under new name. At least not pebbles.jpg

**To Reproduce**
1. Go to https://proton.oli.cmu.edu/sections/sommarprogrammering/page/blskrmsbild
2. Download pebbles.jpg (named "stenar.jpg" in the text) to your computer and rename it (for example to "stenar.jpg"): 
![image](https://user-images.githubusercontent.com/53688217/119979476-51607e80-bfbb-11eb-89b0-09966cdd179a.png)

3. Go to https://proton.oli.cmu.edu/project/kth_swedish_cs101/resource/blskrmsbild/activity/image_coding_activity_c67d2
4. Click on Add Image
5. Select stenar.jpg on your computer. 
6. Nothing is uploaded, but pebbles.jpg is now selected.

**Expected behavior**
Stenar.jpg should be uploaded and selected.

**Screenshots**
![image](https://user-images.githubusercontent.com/53688217/119979822-bae08d00-bfbb-11eb-98ff-82b8597928c3.png)

**Environment (please complete the following information):**
 - OS: macOS 11.4
 - Browser Chrome
 - Version Version 90.0.4430.212 

**Additional context**
It would suffice if I could change the name of the file in the gallery. We just want the Swedish version of the course to be able to use Swedish file names as much as possible.

**Comment by darrensiegel** [comment]
This is in part by design: Torus is determining that you are attempting to upload a file whose content matches exactly the content of another uploaded file (it is comparing MD5 hashes). 

I do understand your intent and use case. I am going to edit the title and description to reflect this.

**Comment by czerwinskie** [comment]
Kristine will attempt to reproduce.

**Comment by nguyenkristine** [comment]
I wasn't able to access the links, I got two errors:
First link it said "Section Has Concluded" and the second link said "404 Not Found".

**Comment by joecoolle** [comment]
You should have received an invitation by now.

MVH
/OB1
